sites/example-project/src/pages/ui-components/tabs/+page.md

```diff
@@ -1,6 +1,6 @@
 ## Tabs
 
-<Tabs id="my-tabs">
```

Comment:
Can we add a second set of tabs on this page that demonstrate how to use the colored variant?

Reply:
Sure. Is this okay ? 

![image](https://github.com/evidence-dev/evidence/assets/129554482/714dd594-d4f7-4ec0-85c6-da31859421bb)